modules/compute/disk-encryption-sets/README.md

```diff
@@ -199,11 +199,7 @@ userAssignedIdentities: {
 
 ## Cross-referenced modules
 
-This section gives you an overview of all local-referenced module files (i.e., other CARML modules that are referenced in this module) and all remote-referenced files (i.e., Bicep modules that are referenced from a Bicep Registry or Template Specs).
```

Comment:
That's a bit unfortunate as there actually still is a (probably redundant) reference. I wonder why it wats to throw this out.

Reply:
Weird I agree, the cross reference is in a nested module, not sure if that is the reason. I'll close this PR, clearly that is not the way we intend to solve it, but I fear some more troubleshooting is needed on the get cross referenced utility